entropylab/results/dashboard/dashboard.py

```diff
@@ -200,4 +219,16 @@ def activate_last_plot_tab_when_tabs_are_changed(children):
             return children[last_tab]["props"]["tab_id"]
         return 0
 
+    @_app.callback(
+        Output("aggregate-clipboard", "content"),
+        Input("aggregate-clipboard", "n_clicks"),
+        State("aggregate-graph", "figure"),
+        prevent_initial_call=True,
+    )
+    def copy_aggregate_data_to_clipboard_as_python_code(_, figure):
+        _dict = list(
+            map(lambda d: {d["name"]: {"x": d["x"], "y": d["y"]}}, figure["data"])
```

Comment:
the format here is python lists, do we want it in numpy?

@galwiner

Reply:
Yes, numpy is the way to go. 
for small data sets it will not make a performance difference, but numpy gives users the functionality they want

Reply:
@galwiner do you mean that if the user used regular arrays then the copy to clipboard code should transform them into numpy ndarrays?

Reply:
Resolved below